src/main/java/thePackmaster/cards/creativitypack/Souvenirs.java

```diff
@@ -0,0 +1,45 @@
+package thePackmaster.cards.creativitypack;
+
+import basemod.helpers.CardModifierManager;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.tempCards.Insight;
+import com.megacrit.cardcrawl.cards.tempCards.Safety;
+import com.megacrit.cardcrawl.cards.tempCards.Smite;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.helpers.CardLibrary;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.FlexibleDiscoveryAction;
+import thePackmaster.cardmodifiers.creativitypack.DrawCardModifier;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import java.util.ArrayList;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Souvenirs extends AbstractCreativityCard {
+
+    public final static String ID = makeID(Souvenirs.class.getSimpleName());
+
+    public Souvenirs() {
+        super(ID, 1, CardType.SKILL, CardRarity.COMMON, CardTarget.NONE);
```

Comment:
This card seems very powerful. Maybe make it unnate? That way it starts on the bottom of the draw pile